x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheMetrics.java

```diff
@@ -8,14 +8,35 @@
 package org.elasticsearch.blobcache;
 
 import org.elasticsearch.telemetry.TelemetryProvider;
+import org.elasticsearch.telemetry.metric.DoubleHistogram;
 import org.elasticsearch.telemetry.metric.LongCounter;
 import org.elasticsearch.telemetry.metric.LongHistogram;
 import org.elasticsearch.telemetry.metric.MeterRegistry;
 
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
 public class BlobCacheMetrics {
+    public static final String CACHE_POPULATION_REASON_ATTRIBUTE_KEY = "cachePopulationReason";
+    public static final String SHARD_ID_ATTRIBUTE_KEY = "shardId";
```

Comment:
Let's use snake case for these attribute names as stated in the [guideline](https://github.com/elastic/elasticsearch/blob/main/modules/apm/NAMING.md#metrics-naming-guidelines). We have not been consistent in all places. But we should do so in future.

Reply:
Done